packages/office-ui-fabric-react/src/components/Popup/Popup.tsx

```diff
@@ -81,4 +81,12 @@ export class Popup extends BaseComponent<IPopupProps, {}> {
         break;
     }
   }
+
+  private _onFocus() {
```

Comment:
In this case it's a native jQuery control we wrap in React, so we have no choice but to get native events to work.

Thanks for the review!